recipe/meta.yaml

```
@@ -1,14 +1,15 @@
 {% set name = "openquake.engine" %}
-{% set version = "3.19.0" %}
+{% set version = "3.21.0" %}
+{% set python_min = "3.7" %}
```

The minimum numba required is not installable on Python 3.7.

```suggestion
{% set python_min = "3.8" %}
```

---

Thanks. I will fix this.